packages/@okta/vuepress-site/docs/guides/implement-password/use-flow/index.md

```diff
@@ -2,9 +2,7 @@
 title: Use the Resource Owner Password Flow
 ---
 
-Before you can begin this flow, you will have to collect the user's password in a manner of your choosing.
-
-Once you have collected the credentials, all that is required is a single API call to the `/token` endpoint:
+Before you can begin this flow, collect the user's password in a manner of your choosing. Once you have collected the credentials, these are then passed through Basic Auth in the request to your [Custom Authorization Server's](/docs/concepts/auth-servers/#custom-authorization-server) `/token` endpoint:
```

Comment:
Before you can begin this flow, you will have to collect the user's password in a manner of your choosing. Once you have collected the credentials, all that is required is a single API call to the [authorization server's](https://developer.okta.com/docs/concepts/auth-servers/) `/token` endpoint. If you are using the default Okta authorization server, then your request would look something like this:

Reply:
I have updated - I also have changed "default Okta authorization server" to default Custom Authorization Server" as there is ongoing confusion with that - so typically clarify "default Custom Authorization Server".